src/test/java/BaseTest.java

```diff
@@ -51,17 +51,19 @@ public void login(String email, String password) throws InterruptedException {
         clickSubmit();
     }
 
+
+    public static void providePassword(String password) {
+        WebElement passwordField = driver.findElement(By.cssSelector("[type='password']"));
+        passwordField.clear();
+        passwordField.sendKeys(password);
```

Comment:
You got this problem, please correct it. 

![image](https://user-images.githubusercontent.com/104761482/211262121-ab267ac1-3b18-4ea1-990f-10a02872ebae.png)

![image](https://user-images.githubusercontent.com/104761482/211262154-c0e6357e-86b5-4e81-868e-dad7a64f66a8.png)